apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -61,8 +92,19 @@ export default class VirtualizedDetailView extends Component {
           </span>
         )}
         {(rowIndex === 0 && columnIndex >= 1) && (
-          <div style={progressStyles.lessonNumberHeading}>
-            {columnIndex}
+          <div style={styles.lessonHeaderContainer}>
+            <div style={styles.numberHeader}>
+              {columnIndex}
+            </div>
+            {(columnWidths[columnIndex] > PROGRESS_BUBBLE_WIDTH) &&
```

Comment:
I have two checks because you can't have adjacent components without a wrapper in react, and I need the arrows to not have a wrapper.